src/csg_tree.cpp

```diff
@@ -604,10 +601,8 @@ std::vector<std::shared_ptr<CsgNode>> &CsgOpNode::GetChildren(
 
   if (forceToLeafNodes && !impl->forcedToLeafNodes_) {
     impl->forcedToLeafNodes_ = true;
-    for_each(impl->children_.size() > 1 && !ManifoldParams().deterministic
-                 ? ExecutionPolicy::Par
-                 : ExecutionPolicy::Seq,
-             impl->children_.begin(), impl->children_.end(), [](auto &child) {
+    for_each(ExecutionPolicy::Par, impl->children_.begin(),
```

Comment:
Changing this to `Seq` caused about 75% of the performance regression I was seeing. Even just the `impl->children_.size() > 1` check was causing noticeable perf regression.